scipy/ndimage/_morphology.py

```diff
@@ -2161,6 +2161,81 @@ def distance_transform_cdt(input, metric='chessboard', return_distances=True,
         Returned only when `return_indices` is True, and `indices` is not
         supplied.
 
+    See Also
+    --------
+    distance_transform_edt : Fast distance transform for euclidean metric
+    distance_transform_bf : Distance transform for different metrics using
+                            a slower brute force algorithm
+
+    Notes
+    -----
+    The distance transform replaces every foreground pixel (pixel value
+    > 0) by its distance to the closest background pixel (pixel value 0).
```

Comment:
To me, this seems like a duplication of the information given on lines 2109-2111